src/components/ScholarshipsMadeSimpleGrid.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import logopic from '../img/blank.png';
+import LoginButton from './LoginButton';
+
+function ContentDiplay(user) {
+  if (user === 0) {
```

Comment:
1. Ah ok its because initially I was passing in `value` which was a number, now I changed it to be `user === 'Student'`.
```javascript
<TabPanel value={value} index={0}>
        <ScholarshipsMadeSimpleGrid user={value} />
```
Now it is 
```javascript
<TabPanel value={value} index={0}>
        <ScholarshipsMadeSimpleGrid user={'Student'} />
```

2. Ok right similar to the `loginButton` display except this is on the `<Typography>`. The change was made thank for providing the ternary operator